_includes/careers/careers_benefits.html

```diff
@@ -10,35 +10,35 @@ <h3 class="section-subheading text-muted">We believe that the best work is done
         <div class="row text-center">
             <div class="col-md-4">
                 <img src="/img/shared/icon-cactus.svg" alt="A house plan for your home office" class="icon icon-laptop" />
-                <h4 class="service-heading">Remote-first culture</h4>
-                <p class="text-muted"> Flexible working hours, near-total freedom to manage your own schedule</p>
+                <h4 class="service-heading">Remote-First Culture</h4>
+                <p class="text-muted">Flexible working hours, near-total freedom to manage your own schedule.</p>
             </div>
             <div class="col-md-4">
                 <img src="/img/shared/icon-chair.svg" alt="A seat in the office for those who socialise" class="icon icon-laptop" />
-                <h4 class="service-heading">An optional hot desk</h4>
-                <p class="text-muted">Space in our Bangsar South office for use if working from home doesn’t appeal to you</p>
+                <h4 class="service-heading">Desk in Our Office</h4>
+                <p class="text-muted">Whether fixed or hot, there's always a space for you if WFH isn't always your thing.</p>
             </div>
             <div class="col-md-4">
                 <img src="/img/shared/icon-sandwich.svg" alt="A sandwich with lettuce, cheese and tomatoes" class="icon icon-laptop" />
-                <h4 class="service-heading">Free lunch at the office</h4>
-                <p class="text-muted">No need to pack a lunch for the office, we've got that covered</p>
+                <h4 class="service-heading">Free Lunch!</h4>
+                <p class="text-muted">If you're coming to the office for the day, lunch is on us.</p>
             </div>
         </div>
         <div class="row text-center">
             <div class="col-md-4">
                 <img src="/img/shared/icon-car.svg" alt="A new Macbook Pro" class="icon icon-laptop" />
```

Comment:
@monsteronfire I guess we'll need a different icon here 😁

Reply:
@calumhalcrow I was just going to ask when I saw your message on Slack. I'll take a look and update it, accordingly.

Reply:
Icon has been updated.